packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -127,6 +128,11 @@ export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
           {items &&
             items.map((item, i) => <SupportListItem item={item} key={i} />)}
         </List>
+        <Box display="flex" justifyContent="flex-end">
```

Comment:
While waiting for UX feedback, does using `DialogActions` instead work here? Feeling that it's a bit more specific and semantically correct

Reply:
Yeah, works too. I'm agree with you that would be more specific, will commit the changes for that.